webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -137,6 +137,8 @@ public async Task<string> ExtractUserIntentAsync(SKContext context)
             settings: this.CreateIntentCompletionSettings()
         );
 
+        context.Variables.Set("userIntentExtractionTokenUsage", Utilities.GetTokenUsage(result).ToString(CultureInfo.InvariantCulture));
```

Comment:
nit: use an enum to define the variable names.

Reply:
C# doesn't easily support string enums so I used a dict